CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 2.5.0
+
+- Scorecard details page revamp
+- Scorecard service page revamp
+- Initiative page revamp
```

Comment:
```suggestion
- UI overhaul of Scorecard pages with better visualization of rules & levels for Scorecards 
- UI overhaul of Initiative pages 
- Add support for viewing rule exemptions for Scorecards 
- Copy to CSV button for Scorecard scores 
```

@FoodProduct wdyt?

Reply:
Let's use command-tone or whatever it's called to be consistent
```suggestion
- Overhaul the UI of Scorecard pages with better visualization of rules & levels
- Overhaul the UI of Initiative pages
- Add a "Copy to CSV" button for exporting Scorecard scores
```